applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element.cpp

```diff
@@ -33,388 +33,330 @@
 #include "utilities/normal_calculation_utils.h"
 
 namespace Kratos {
-	namespace Testing {

```

Comment:
The entire file has been messed up. I suggest reverting to the master version to only add the necessary changes.

Reply:
Right. The suggested modification are done